src/leader.py

```diff
@@ -152,6 +163,62 @@ def lead(self):
             self.turn(code, position)
 
 
+    def demo_circle(self):
+        """
+        Controls the leader vehicle in a circle for demo purposes.
+
+        Drives the leader vehicle in a continuous circle till script is
+        terminated.
+        """
+        while True:
+            self.fw.turn(DEMO_LEFT_TURN)
+            self.bw.speed(DEMO_SPEED)
+            self.bw.forward()
+
+
+    def demo_line(self):
```

Comment:
Do we even need this one actually? It just demonstrates an oval basically. The circle is a more intensive demonstration of this. We can show straight line driving ourselves.